ocfweb/docs/docs/staff/procedures/accounts/renaming.md

```diff
@@ -0,0 +1,32 @@
+[[!meta title="Rename an account"]]
+
+We should sparingly rename a user account, and only do so when it is
+necessary (e.g. typo in the username).
+
+To rename an individual account, `sorry` the account with the old name and
+make a `note` of why you are changing the username. To run `sorry` you need
+to be a root user. Follow the prompts of the script.
+
+```
+sorry username_to_rename
```

Comment:
```suggestion
sudo sorry username_to_rename
```

This draws the distinction between running `sorry` as root and `note` as your user.